src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ZipStatusController.java

```diff
@@ -23,8 +27,17 @@ public ZipStatusController(ZipFileStatusService service) {
     }
     // endregion
 
-    @GetMapping
-    public ZipFileStatus findByFileName(@RequestParam("name") String fileName) {
-        return service.getStatusFor(fileName);
+    @RequestMapping
+    public ResponseEntity<List<ZipFileStatus>> findByFileNameOrDcn(@RequestParam Map<String,String> params) {
+        String fileName = params.get("name");
```

Comment:
param name should be 'fileName'

Reply:
It was "name" before so I didn't change it.

Reply:
you are right